.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: python
+
+python:
+  - "2.7"
+  - "3.5"
+
+virtualenv:
+  system_site_packages: true
```

Comment:
I think `system_site_packages` is only needed when we need to use globally installed python packages. I don't see a need for it here since we're not installing any global packages?